src/dialogs/examples/teams/UpdateCardMsgSetupDialog.ts

```diff
@@ -3,23 +3,38 @@ import { TriggerActionDialog } from "../../../utils/TriggerActionDialog";
 import { DialogIds } from "../../../utils/DialogIds";
 import { DialogMatches } from "../../../utils/DialogMatches";
 import { Strings } from "../../../locale/locale";
+let config = require("config");
```

Comment:
I think we normally do 
import * as config from "config"

Reply:
Done.